### PR TITLE
Maintenance Page: State, Mutations and Actions added

### PR DESCRIPTION
**Pull request by JuanJosePG**

### CODE REVIEW

File: src/store/main-store/MainState.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export default class MainState {
     public testData: boolean = true;
     public currentLanguage: string = 'en';
+    public isMaintenance: boolean = false;
```
Comment (gonzalo-plaza) [inline src/store/main-store/MainState.ts:4]: @JuanJosePG the value must be true, because in the app, you use the maintenance component if isMaintenance

Comment (JuanJosePG) [reply src/store/main-store/MainState.ts:4]: But if I set this value to true, the maintenance page is active and we didn't see the login view, Im right?

Comment (gonzalo-plaza) [reply src/store/main-store/MainState.ts:4]: Yes you are right, sorry I have confused.

**Comment by gonzalo-plaza** [comment]
@JuanJosePG For me this is okay to can be merged, but before we must fix the duplicate commit in the another PR.

**Comment by Andresws12** [comment]
@JuanJosePG I'm not sure what is the correct PR, please remove the unnecessary PR

**Pull request closed without merging**